### ISSUE TITLE
Redwood City Police Media Log PDFs are rendered as blank, in PDF.js (with "Warning: Error during JPEG image loading" and "Security Error: Content at resource://pdf.js/web/viewer.html may not load data from blob:null/[...]")

### ISSUE THREAD

**Issue by dholbert** [mozilla/pdf.js#6017]
STR:
Try to load any of the PDFs linked from http://www.redwoodcity.org/police/information/ml/ , e.g. http://www.redwoodcity.org/police/medialog/2015/medialog_20150512.pdf

ACTUAL RESULTS:
The PDF loads with blank pages. Additionally, the browser console shows the following:
```
PDF 741ae08b51a402b4fb0c9675f43086f7 [1.4 Eastman Kodak Company / -] (PDF.js: 1.1.138) viewer.js:6403
Warning: Error during JPEG image loading pdf.js:235
Security Error: Content at resource://pdf.js/web/viewer.html may not load data from blob:null/1fcb8fba-9580-431a-87b9-8bbad9b72921.
Warning: Dependent image isn't ready yet pdf.js:235
Security Error: Content at resource://pdf.js/web/viewer.html may not load data from blob:null/ee49050a-30aa-4df5-aea7-f35333a51deb.
Warning: Error during JPEG image loading pdf.js:235
Warning: Dependent image isn't ready yet pdf.js:235
```

EXPECTED RESULTS:
Evince (linux PDF viewer) renders a non-blank PDF, with what looks like a scanned document image. PDF.js should show that same rendering.

Reproducing using Nightly 41.0a1 (2015-05-14), with built-in PDF.js, as well as the latest https://mozilla.github.com/pdf.js/extensions/firefox/pdf.js.xpi  (version 1.1.138)

**Comment by Snuffleupagus** [comment]
This is a duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1164199.

**Comment by dholbert** [comment]
Looks like it, thanks.

(When I've filed PDF Viewer bugs on Bugzilla before, I've been told it's better to file them here instead. Is that still the case?)

**Comment by Snuffleupagus** [comment]
> (When I've filed PDF Viewer bugs on Bugzilla before, I've been told it's better to file them here instead. Is that still the case?)

Yes; if the bug is present in the latest version of PDF.js, GitHub is usually preferred.

Since we (unfortunately) have two bug trackers, my post was simple intended to help us track this issue across the two. (I'm sorry if it came off as terse).

**Comment by dholbert** [comment]
No worries; just wanted to make sure I wasn't behind the times. :) Thanks for the triage!

**Comment by Lord-of-Darkness** [comment]
Desarrolladores y líderes de Mozilla, que son travestis culo!

Alguien debe colgar ustedes para arriba en sus propias bolas y dejar que su perro muerda su caricatura de una polla!

Posteriormente deben ponerlo tiempo que sea necesario por la garganta hasta que traga a sí mismos en el suicidio!

Ustedes Mozilla está ... Estúpido de mierda coños!
--------------------------------------------------------
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!

**Comment by Lord-of-Darkness** [comment]
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!
.....................................................
You can lick my penis and my golden eggs and kiss my ass son of a bitch with a personality disorder from the depths of the abyss of hell from the abysses of Mozilla!

Fuck you and your mother you autistic son of a bitch, you will lick Putin eggs, fascist fucking faggot!

The shit out of your dog, your cat and your dirty faggot asshole mountain goat!

Suck cocks Mozilla and together go and suck Google ass full of shit!

Shit in your knee you cock suckers! And that damn Australis Chrome clone of shit! Mozilla gay homosexual lesbian fucking bastards from hell!
....................................
Do you love the pain that you do? And to do a favour... I'm going to do that!

I kicked her so hard failure golden eggs until they are ringing and sing you all the melodies of forward and back pain in an instant! I am the boy Nutcracker who cracks your are crazy nutheads!

And now go to the stupid shit shit cones!
....................................................................
Was it expected?

Oh, Yes, that was!

A movement such as the Mozilla!

So happy that I never upgraded to Firefox 29 and beyond!

Let me tell you something Mozilla developers! If I would be still using your latest crap-ware that really my mission would at this time to find out where you live and then drill him often in Golden's failure until you analyzes and understands how to threaten power users in the correct way!

Without a doubt, has been said before - but may not be repeated enough... You are...

Stupid cones!

**Comment by Lord-of-Darkness** [comment]
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!
.....................................................
You can lick my penis and my golden eggs and kiss my ass son of a bitch with a personality disorder from the depths of the abyss of hell from the abysses of Mozilla!

Fuck you and your mother you autistic son of a bitch, you will lick Putin eggs, fascist fucking faggot!

The shit out of your dog, your cat and your dirty faggot asshole mountain goat!

Suck cocks Mozilla and together go and suck Google ass full of shit!

Shit in your knee you cock suckers! And that damn Australis Chrome clone of shit! Mozilla gay homosexual lesbian fucking bastards from hell!
....................................
Do you love the pain that you do? And to do a favour... I'm going to do that!

I kicked her so hard failure golden eggs until they are ringing and sing you all the melodies of forward and back pain in an instant! I am the boy Nutcracker who cracks your are crazy nutheads!

And now go to the stupid shit shit cones!
....................................................................
Was it expected?

Oh, Yes, that was!

A movement such as the Mozilla!

So happy that I never upgraded to Firefox 29 and beyond!

Let me tell you something Mozilla developers! If I would be still using your latest crap-ware that really my mission would at this time to find out where you live and then drill him often in Golden's failure until you analyzes and understands how to threaten power users in the correct way!

Without a doubt, has been said before - but may not be repeated enough... You are...

Stupid cones!

**Comment by Lord-of-Darkness** [comment]
Mozilla team! You betray the spirit of code open and another time! Fortunately, more and more people are realizing that kind of hypocrites you are!

You are never going to be the number one in the browser market! Stop thinking you could be a leader, never exceed Chrome! So stop running behind Google and again focus on geeks and power users, bring personalization back, stop adding more "Chrome parity"! Add place better "Vivaldi parity"-not from the development team, you can learn how users should be treated! With respect and with the aim of satisfying your needs instead of fleeing as it operates and betray its user base kernel and other time and again and again...

You is the Mozilla developers have a brain at all, or are you just treacherous opportunists in Google Paylist? Australis is an imitation without value of Google user interface - you betray your Open Source origins with the addition of DRM and ads... well, another step to sell, the same as add mess owner as of Pocket!

You're traitors to all related open source! Shame on you and your developers for Mozilla "out of this world" "leaders" with less than half of a brain!

I only say vivaldi.com - palemoon.org - otter-browser.org or seamonkey-project.org

**Comment by Lord-of-Darkness** [comment]
Mozilla team! You betray the spirit of code open and another time! Fortunately, more and more people are realizing that kind of hypocrites you are!

You are never going to be the number one in the browser market! Stop thinking you could be a leader, never exceed Chrome! So stop running behind Google and again focus on geeks and power users, bring personalization back, stop adding more "Chrome parity"! Add place better "Vivaldi parity"-not from the development team, you can learn how users should be treated! With respect and with the aim of satisfying your needs instead of fleeing as it operates and betray its user base kernel and other time and again and again...

You is the Mozilla developers have a brain at all, or are you just treacherous opportunists in Google Paylist? Australis is an imitation without value of Google user interface - you betray your Open Source origins with the addition of DRM and ads... well, another step to sell, the same as add mess owner as of Pocket!

You're traitors to all related open source! Shame on you and your developers for Mozilla "out of this world" "leaders" with less than half of a brain!

I only say vivaldi.com - palemoon.org - otter-browser.org or seamonkey-project.org

**Comment by Lord-of-Darkness** [comment]
Mozilla team! You betray the spirit of code open and another time! Fortunately, more and more people are realizing that kind of hypocrites you are!

You are never going to be the number one in the browser market! Stop thinking you could be a leader, never exceed Chrome! So stop running behind Google and again focus on geeks and power users, bring personalization back, stop adding more "Chrome parity"! Add place better "Vivaldi parity"-not from the development team, you can learn how users should be treated! With respect and with the aim of satisfying your needs instead of fleeing as it operates and betray its user base kernel and other time and again and again...

You is the Mozilla developers have a brain at all, or are you just treacherous opportunists in Google Paylist? Australis is an imitation without value of Google user interface - you betray your Open Source origins with the addition of DRM and ads... well, another step to sell, the same as add mess owner as of Pocket!

You're traitors to all related open source! Shame on you and your developers for Mozilla "out of this world" "leaders" with less than half of a brain!

I only say vivaldi.com - palemoon.org - otter-browser.org or seamonkey-project.org

**Comment by timvandermeij** [comment]
Fixed temporarily by #6024 and will be tracked on Bugzilla for a final fix.
